forwards_paper.tex

```diff
@@ -1179,4 +1233,15 @@ \section*{Supplementary Information}
 neutral variation over the run times tracking the pedigree.}
 \end{figure}
 
+\begin{figure}
+    \includegraphics[]{sims/memuse}
+    \caption{\label{sfig:ramsel}Peak RAM use in gigabytes (GB) for simulations with selection.  The plots are from the
+        same data as in Figure \ref{fig:runtimes_selection}.}
+\end{figure}
+
+\begin{figure}
+    \includegraphics[]{sims/memuse_nosel}
+    \caption{\label{sfig:ramnosel}Peak RAM use in gigabytes (GB) for simulations without selection.  The plots are from
+    the same data as in Figure \ref{sfig:rawspeed_nosel}.}
```

Comment:
Add: Note vastly different y-axis scales between sims with netural mutations and with pedigree tracking